index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Birds?</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./normalize.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <main>
+    <body>
+      <header>
+        <section class="nav-block">
+          <div class="site-name">
+            <img src="./icons/burb.png" class="logo">
```

Comment:
Although these images are small and next to them is the text that explains what/where they go, it's still best practice to include alt text.

Reply:
I almost did that but then thought otherwise so thank you for re-stating that this is the right way to do it!